src/Pim/Bundle/DataGridBundle/Normalizer/Product/FileNormalizer.php

```diff
@@ -17,13 +17,19 @@ class FileNormalizer implements NormalizerInterface
      */
     public function normalize($file, $format = null, array $context = [])
     {
+        $fileData = null;
```

Comment:
both would work I think, but I'd rather have a data that is null when a product's value data is null than having an empty array.
The storage and index normalization format also puts null. (It's a DataGridBundle/Normalizer the purpose was to have a more defensive code here but I get your comment).